src/main/java/io/javalin/Javalin.java

```diff
@@ -165,7 +166,8 @@ public JavalinServlet createServlet() {
             maxRequestCacheBodySize,
             prefer405over404,
             singlePageHandler,
-            new JettyResourceHandler(staticFileConfig, jettyServer, ignoreTrailingSlashes)
+            new JettyResourceHandler(staticFileConfig, jettyServer, ignoreTrailingSlashes),
+            exceptionOnRequestCacheIsSmall
```

Comment:
Did you check the EmbeddedJavalin class and if it requires any changes?

Reply:
initially I wanted to make this parameter (to enable exception) unobtrusive. but I guess it's better to make it as normal parameter of javalin. 
I added it to createServlet of EmbeddedJavalin